jax/_src/interpreters/mlir.py

```diff
@@ -663,7 +683,7 @@ def _get_mem_kind(s: Optional[XLACompatibleSharding]) -> Optional[str]:
 
 def lower_jaxpr_to_module(
     module_name: str,
-    jaxpr: core.ClosedJaxpr,
+    jaxpr: core.ClosedJaxpr, *,
```

Comment:
put the `*` on a new line please? It's very easy to miss given that all parametes are on a new line.

Reply:
Done